Add release notes for 11.39 (#558)
mondoohq/docs: releases/2025-01-28-mondoo-11.39-is-out.md

---
slug: mondoo-11.39-is-out/
title: Mondoo 11.39 is out!
description: Announcing the 11.39 release of Mondoo with cases and search in workspaces, improved asset risk exposure, and more!
authors: [tim]
image: /img/releases/2025-01-28-mondoo-11.39-is-out/risk_insights.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.39 is out! This release includes cases and search in workspaces, improved asset risk exposure, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### Create cases in workspaces

> Turn prioritized findings into action with cases within your workspaces. Create cases directly from surfaced findings and track them in individual workspaces or the parent space for consolidated management of work.

Reviewer: ```suggestion
Turn prioritized findings into action: You can now create a case from within a workspace. Track cases in the workspaces where you created them, or in the parent space for consolidated management.
```

![Cases within workspaces](/img/releases/2025-01-28-mondoo-11.39-is-out/cases.png)

### Search for assets and findings in workspaces

> Quickly track down assets, CVEs, checks, or policies in your workspaces with global search within workspaces.

Reviewer: ```suggestion
Quickly track down assets, CVEs, checks, or policies with the new global search within workspaces.
```

![Search within workspaces](/img/releases/2025-01-28-mondoo-11.39-is-out/search.png)

## 🧹 IMPROVEMENTS

### Risk insights on assets

> Quickly assess asset risks with the updated Risk Insights section, displaying total counts of high and critical findings and vulnerabilities. Click any section to drill in particular findings to fix.

Reviewer: ```suggestion
Quickly assess asset risks with the updated Risk Insights section, displaying total counts of high and critical findings and vulnerabilities. Click any section to drill down into particular findings to fix.
```

![Asset page with risk insights](/img/releases/2025-01-28-mondoo-11.39-is-out/risk_insights.png)

## 🐛 BUG FIXES AND UPDATES

- Fix asset technology fields on VMware and OCI assets.
- Add OCI to the list of technology values available for workspace creation.
- Fix incorrect CVSS scores displayed in CVE page score boxes.
- Ensure that positive risk factors display in green.
- Display the save button on the security model settings tab even if the advanced settings aren't selected.
- Update the workspaces selection filtering to move "Kind" under infrastructure.
- Fix failures generating some exports.

> - Support certificate based authentication for the Microsoft Defender integration.

Reviewer: ```suggestion
- Support certificate-based authentication for the Microsoft Defender integration.
```

- Add the version number to the PCI DSS compliance framework to make it more clear which version is included.
- Fix the application of risk factors on Debian-based systems.
- Show correct timestamps for export and import integrations.
- Fix missing asset titles in space and organization search.

> - Improve the display of Ansible, Bash, and PowerShell based remediation steps for advisories.

Reviewer: ```suggestion
- Improve the display of Ansible-, Bash-, and PowerShell-based remediation steps for advisories.
```

> - Improve calculation of risk scores on assets.

Reviewer: ```suggestion
- Improve calculation of asset risk scores.
```
